### PR TITLE
Image Component

### PR DESCRIPTION
**Pull request by khyatibaria**
### Image Component
Please Review it
image

### CODE REVIEW

**Comment by AtulHande03a** [comment]
looks good to merge, just a suggestion to add netlify link also for preview

**Review by nyyshaaa — COMMENTED** [review]
Good to merge!

File: image/image.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  width: 300px;
+  height: 300px;
+}
+
+.img-responsive {
+  width: 100%;
+  height: auto;
+}
+
+.img-cornerrounded {
```
Comment (zabihshaik) [inline image/image.css:13]: Better to have separation for corner-rounded or name it roundedcorner

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. 
Just check the single comment please.

**Pull request merged**